imsApp/Db/ims_long.db

```diff
@@ -747,3 +674,33 @@ record(ao, "$(MOTOR):COPY_ACT") {
   field(OMSL, "closed_loop")
 }
 
+record(stringout, "$(MOTOR):SAVED_STR") {
+  field(DESC, "Saved Raw Position")
```

Comment:
They are the same value, different formats (one string, one int).  I don't think it's important to make this distinction, especially since we probably won't look at this field anyway.